### ISSUE TITLE
Weekly Privacy Statement prompts

### ISSUE THREAD

**Issue by asutherland** [Pernosco/pernosco#14]
About once a week I end up getting the Privacy Statement prompt.  This seems consistent with the "pernosco-consentmozilla" cookie having an Expires set a week in the future, roughly the same as the auth cookies (although they live for an extra 10 minutes).  The document says it was last updated October 3rd.  Do I need to consent to it every week?

**Comment by rocallahan** [comment]
We're trying to set max-age to a year:
```
   document.cookie = "pernosco-consent" + name + "=" + version + ";path=/;max-age=31536000";
```
I don't know why that isn't working.

**Comment by asutherland** [comment]
Oh, Firefox limits JS-set cookies to a week.
https://searchfox.org/mozilla-central/rev/91f6c02fcf4c16f78fdc4417f61f192688294066/netwerk/cookie/nsCookieService.cpp#4275 is where it's applied.

https://searchfox.org/mozilla-central/source/__GENERATED__/modules/libpref/init/StaticPrefList_privacy.h#232 is the pref which is:
```cpp
ALWAYS_PREF(
  "privacy.documentCookies.maxage",
   privacy_documentCookies_maxage,
   privacy_documentCookies_maxage,
  uint32_t, 7 * 24 * 60 * 60
)
```

Maybe the server can do it?

**Comment by asutherland** [comment]
Alternately, I hear great things about IndexedDB and localStorage! ;)

**Comment by rocallahan** [comment]
I think we need to use cookies here so that our HTTP router can route based on the values.

**Comment by khuey** [comment]
Fixed in 99556db37c4ef773d42856e46bc2ee4f90613e86
